### DISCUSSION TITLE
人工智能学习路线与资料 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#68] [Announcements]
# 人工智能学习路线与资料 | HackWay技术学习路线

在2022年前，人工智能和机器学习（即AI+D）还是六大计算机名校CS专业本科必修的课程，但是2022年后，AI+D则渐渐成为垂直领域的选修课，但是在2023年后又被列为必修。人工智能的学习具有相当高的门槛，你需要在学习前具备一定数学（离散数学、微积分、线性代数、概率统计）、编程（至少掌握Python）、数据结构与算法的能力，如果你没有这样的基础，请不要跟风学习。值得说明的是，名校里的人工智能课程侧重于理论的学习，对于一般的程序员人工智能的理论仍然值得来哦接。

https://hackway.org/docs/ai/intro

**Comment by SoftSir** [comment]
请问楼主有推荐的学习路线吗
